From 0247c95a292308beb38ed960000e3dd22faa83d1 Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Wed, 26 Dec 2018 13:45:08 +0100
Subject: [PATCH 1/9] selinux: move vendor_camera_prop from
 device/qcom/sepolicy

Snap wants to read vendor specific camera props which are not just a qcom thing.
To be able to grant Snap the right to read those props without breaking non-qcom devices
the definition for vendor_camera_prop should be moved to global lineage ruleset.

Change-Id: Ic3cbe2f3074ab44d1ccfdd36d1c8d0cfeaa8bb43
---
 common/public/property.te       | 1 +
 common/vendor/property_contexts | 3 +++
 2 files changed, 4 insertions(+)
 create mode 100644 common/vendor/property_contexts

diff --git a/common/public/property.te b/common/public/property.te
index 81735e0..168cd00 100644
--- a/common/public/property.te
+++ b/common/public/property.te
@@ -1 +1,2 @@
 type sf_lcd_density_prop, property_type;
+type vendor_camera_prop, property_type;
diff --git a/common/vendor/property_contexts b/common/vendor/property_contexts
new file mode 100644
index 0000000..c042929
--- /dev/null
+++ b/common/vendor/property_contexts
@@ -0,0 +1,3 @@
+#vendor-camera
+vendor.camera.aux.packagelist      u:object_r:vendor_camera_prop:s0
+persist.vendor.camera.privapp.list u:object_r:vendor_camera_prop:s0
-- 
2.17.1

